app/controllers/api/v0/versions_controller.rb

```diff
@@ -15,7 +15,8 @@ def show
     render json: {
       links: {
         page: api_v0_page_url(@version.page),
-        previous: @version.previous && api_v0_page_version_url(@version.page, @version.previous)
+        previous: @version.previous && api_v0_page_version_url(@version.page, @version.previous),
+        next: @version.next && api_v0_page_version_url(@version.page, @version.next)
```

Comment:
This works great, but would you mind changing it to use `api_v0_version_url()` instead of `api_v0_page_version_url()` for both of these? That simpler route didn’t exist when we originally made this.